### PR TITLE
[CP3.0] Standalone cert-manager

### PR DESCRIPTION
**Pull request by qpdpQ**
This PR plan to do:

- [x] - Add V1 CRDs back to the bundle
- [x] - remove v1alpha1 CRDs
- [x] - remove all the extra functions (including: cert-rotation, v1alpha1 conversion, pod restart)
- [ ] - find a way to make cert-manager always deploy its operand (will open another ticket and a new pr for that)
- [ ] - cluster permission reduction （most part of that are done）

### CODE REVIEW

File: controllers/resources/util.go [hunk 1]
```diff
@@ -57,6 +57,7 @@ func GetImageID(imageRegistry, imageName, defaultImageVersion, imagePostfix, env
 func GetDeployNamespace() string {
 	// If running locally or installMode is All-Namespaces use "ibm-common-services"
 	ns, _ := os.LookupEnv("OPERATOR_NAMESPACE")
+	// Issue
```
Comment (bitscuit) [inline controllers/resources/util.go:60]: what is this?

Comment (qpdpQ) [reply controllers/resources/util.go:60]: I will investigate on why `ns, _ := os.LookupEnv("OPERATOR_NAMESPACE")` could return ""

Comment (qpdpQ) [reply controllers/resources/util.go:60]: This is what I saw in the test
image
image

Comment (bitscuit) [reply controllers/resources/util.go:60]: We will look at this issue in the suggest-namespace PR

**Review by bitscuit — APPROVED** [review]

**Pull request merged**